Pushbullet/Api/ServerApiEntryPoints.cs

```diff
@@ -24,11 +25,13 @@ public class TestNotification : IReturnVoid
     class ServerApiEndpoints : IService
     {
         private readonly IHttpClient _httpClient;
+		private readonly IJsonSerializer _jsonSerializer;
         private readonly ILogger _logger;
 
-        public ServerApiEndpoints(ILogger logger, IHttpClient httpClient)
+        public ServerApiEndpoints(ILogger logger, IJsonSerializer jsonSerializer, IHttpClient httpClient)
         {
               _logger = logger;
+			  _jsonSerializer = jsonSerializer;
```

Comment:
You're using tabs it seems. We use 4 spaces.
```suggestion
              _jsonSerializer = jsonSerializer;
```